src/bundle/Resources/public/js/scripts/admin.role_assignment.list.js

```diff
@@ -0,0 +1,10 @@
+(function () {
+    const checkboxes = [...document.querySelectorAll('form[name="role_assignments_delete"] input[type="checkbox"]')];
+    const buttonRemove = document.querySelector('#delete-role-assignments');
+    const toggleButtonState = (event) => {
+        const methodName = checkboxes.some(el => el.checked) ? 'removeAttribute' : 'setAttribute';
+        buttonRemove[methodName]('disabled', true);
+    }
+
+    checkboxes.forEach(checkbox => checkbox.addEventListener('change', toggleButtonState, false));
```

Comment:
The whole code here duplicates the code from `admin.policy.list.js` we can avoid it, by using another selector in the file above.